src/esp/assets/ResourceManager.cpp

```diff
@@ -182,7 +182,8 @@ bool ResourceManager::loadStage(
     const std::shared_ptr<physics::PhysicsManager>& _physicsManager,
     esp::scene::SceneManager* sceneManagerPtr,
     std::vector<int>& activeSceneIDs,
-    bool loadSemanticMesh) {
```

Comment:
I renamed loadSemanticMesh to createSemanticMesh just to match the var name in the header.